packages/patternfly-4/react-core/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Radio/radio.css';
+import { css, getModifier } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface RadioProps extends Omit<React.HTMLProps<HTMLInputElement>, 'label' | 'onChange'> {
+  /** Additional classes added to the radio. */
+  className?: string;
+  /** Id of the radio. */
+  id: string;
+  /** Flag to show if the radio is checked. */
+  isChecked?: boolean;
+  /** Flag to show if the radio is disabled. */
+  isDisabled?: boolean;
+  /** Flag to show if the radio selection is valid or invalid. */
+  isValid?: boolean;
+  /** Label text of the radio. */
+  label?: React.ReactNode;
+  /** Name for group of radios */
+  name: string;
+  /** A callback for when the radio selection changes. */
+  onChange?: (checked: boolean, event: React.FormEvent<HTMLInputElement>) => void;
+  /** Aria-label of the radio. */
```

Comment:
Can you update the description here.  Can we say `Aria label` or  `The aria-label`